doc/man3/EVP_PKEY_CTX_ctrl.pod

```diff
@@ -160,6 +166,17 @@ For maximum compatibility the named curve form should be used. Note: the
 B<OPENSSL_EC_NAMED_CURVE> value was only added to OpenSSL 1.1.0; previous
 versions should use 0 instead.
 
+The EVP_PKEY_CTX_set1_id(), EVP_PKEY_CTX_get1_id() and EVP_PKEY_CTX_get1_id_len()
+are used to manipulate the special identifier field for specific signature
+algorithms such as SM2. The EVP_PKEY_set1_id() sets an ID pointed by B<id> with
+the length B<id_len> to the library. The library maintains the memory management
+stuffs so the caller can safely free the original memory pointed by B<id>. The
```

Comment:
Change this to "The library takes a copy of the id so that the caller can safely free the original memory pointed to by B\<id\>"